tests/QAN/details_test.js

```diff
@@ -117,6 +117,32 @@ Scenario(
   },
 );
 
+Scenario(
+  'PMM-T1790 - Verify that there is any no error on Explains after switching between queries from different DB servers '
+    + '@qan',
```

Comment:
may be add nightly?

Reply:
Does it makes sense since "qan" tag is executed only at nightly job: https://github.com/Percona-Lab/jenkins-pipelines/blob/378539e5e11d398f7872b8988784f4c4f40ebe8a/pmm/pmm2-ui-tests-nightly.groovy#L356?